docs/content/doc/installation/with-docker.en-us.md

```diff
@@ -303,7 +303,23 @@ services:
     - GITEA__mailer__PASSWD="""${GITEA__mailer__PASSWD:?GITEA__mailer__PASSWD not set}"""
 ```
 
-To set required TOKEN and SECRET values, consider using Gitea's built-in [generate utility functions](https://docs.gitea.io/en-us/command-line/#generate).
+To set required TOKEN and SECRET values, ensure you generate new values per installation. The following docker commands will make use of Gitea's built-in [generate utility functions](https://docs.gitea.io/en-us/command-line/#generate).
```

Comment:
Gitea will generate new secrets/tokens for every new installation automatically. So the sentence "ensure you generate new values per installation" seems not telling readers the details.

```suggestion
Gitea will generate new secrets/tokens for every new installation automatically and write them into the app.ini. If you want to set the secrets/tokens manually, you can use the following docker commands to use of Gitea's built-in [generate utility functions](https://docs.gitea.io/en-us/command-line/#generate). Do not lose/change your SECRET_KEY after the installation, otherwise the encrypted data can not be decrypted anymore.
```

Not sure whether my suggestion is right, please help to correct it.